### PR TITLE
LPS-137315 Hide Searchable field when object storage type is selected

### PR DESCRIPTION
**Pull request by mateussgn**
Related issue: https://issues.liferay.com/browse/LPS-137315

### CODE REVIEW

**Comment by mateussgn** [comment]
ci:test:sf

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/fields-sidebar/components/FieldsSidebarSettingsBody.js [hunk 1]
```diff
@@ -44,6 +44,10 @@ const getColumn = ({objectFields}) => ({children, column, index}) => {
 					return <React.Fragment key={index} />;
 				}
 
+				if (objectFields.length && fieldName === 'indexType') {
+					return <React.Fragment key={index} />;
+				}
+
```
Comment (interaminense) [inline modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/fields-sidebar/components/FieldsSidebarSettingsBody.js:50]: @mateussgn,

Could you change the condition? Use something like this:

```
if (objectFields.length && (fieldName === 'repeatable' || fieldName === 'indexType')) {
     return <React.Fragment key={index} />;
}
```

**Review by interaminense — CHANGES_REQUESTED** [review]

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/fields-sidebar/components/FieldsSidebarSettingsBody.js (hunk 1 above)
Comment (interaminense) [reply modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/fields-sidebar/components/FieldsSidebarSettingsBody.js:50]: Also change the comment above the condition

`// Avoid using repeatable field when object storage type is selected`

**Comment by mateussgn** [comment]
ci:test:sf

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
ci:test:relevant

**Review by guilhermedcamacho — APPROVED** [review]

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:test:relevant

**Comment by mateussgn** [comment]
ci:forward

**Pull request closed without merging**